web/build.gradle

```diff
@@ -32,7 +32,7 @@ gretty {
     serverConfigFile = "tomcat-gretty.xml"
 
     jvmArgs = ["-Dcatalina.base=${buildDirectory}/catalina.base",
-               "-Dspring.profiles.active=dev,caching-enabled,wc-security-test,ia-stub,certificate-service-active,hsa-integration-intyg-proxy-service,servicenow-integration",
+               "-Dspring.profiles.active=dev,caching-enabled,wc-security-test,ia-stub,certificate-service-active,hsa-integration-intyg-proxy-service,wc-kundportalen-stub,wc-all-stubs",
```

Comment:
Is it correct that we should use these profiles? Or were the left after testing?

Reply:
Profile set to servicenow-intgration